lerna.js

```diff
@@ -10,7 +10,22 @@ async function depInfo({ location, name }, log) {
   let ignore = files ? folders.filter(elem => files.every(file => !file.startsWith(elem.name))) : folders;
   ignore = ignore.map(x => x.isDirectory() ? `${x.name}/**` : x.name)
 
-  const {dependencies, devDependencies, missing, using} = await checkDeps(location, { ignorePatterns: ignore }, val => val);
+  let {dependencies, devDependencies, missing, using} = await checkDeps(location, { ignorePatterns: ignore }, val => val);
+
+  if (Object.values(using).flat().some(file => 
+    readFileSync(file, 'utf8').toString().includes('const process: NodeJS.Process = require(\'process/\')') ||
```

Comment:
Instead of these 4 cases, I assume we could just check for the presence of `require(\"process/\")`?
This would also allow us to capture the case where `process` is used inline.